packages/synapse-react-client/src/components/SynapseTable/SynapseTableCell/JSON/JSONArrayRenderer.tsx

```diff
@@ -0,0 +1,55 @@
+import { JSONPrimitiveType } from './JSONRendererUtils'
+import React, { useState } from 'react'
+import { Box, Collapse, Typography } from '@mui/material'
+import { JSONPrimitiveRenderer } from './JSONPrimitiveRenderer'
+import ExpandCollapseButton from '../../ExpandCollapseButton'
+
+type JSONArrayRendererProps = { value: JSONPrimitiveType[] }
+
+/**
+ * Handles rendering a JSON array of "primitive" types in a Synapse table cell.
+ * @param props
+ * @constructor
+ */
+export function JSONArrayRenderer(props: JSONArrayRendererProps) {
+  const { value } = props
+  const [expanded, setExpanded] = useState(false)
+  const showCollapseText = value.length > 1
+
+  if (value.length === 0) {
+    return (
+      <Typography variant={'smallText1'} className="SRC-inactive">
+        {'empty array'}
```

Comment:
Is this the expected output?  Or would something friendlier be preferred (like "No items")?

Reply:
Will check with design